environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - pandas>=0.20.0
   - flake8>=3.7.7
   - flake8-docstrings>=1.5.0
-  - black>=22.8.0
+  - black>=22.1.0
```

Comment:
I believe this should be `23.1.0` rather than `22.1.0`, right?

Reply:
@seanlaw 
I think the most updated one is `23.1.0`. I was doing some experiment and I realized that I had to install the most updated version to handle [skip-trailing-comma](https://github.com/psf/black/issues/2135). Otherwise, there was a discrepency between the behavior of black between the local and remote branch. I should have chagned this yml file as well since stumpy install `black 23.1.0` in git actions.